tests/src/FunctionalJavascript/OpenVocabularyFormTest.php

```diff
@@ -0,0 +1,163 @@
+    $assert_session->fieldExists('Description')->setValue($description);
+    $assert_session->elementExists('named', ['fieldset', 'Vocabulary type']);
+    $assert_session->elementExists('named', ['radio', 'Test entities']);
+    $assert_session->elementExists('named', ['radio', 'Tests the info alter hook']);
+    $assert_session->elementExists('named', ['radio', 'Entity test with bundle']);
+    // Verify that the vocabulary type field is required.
+    $this->getSession()->getPage()->pressButton('Save');
+    $assert_session->pageTextContains('Error message Vocabulary type field is required.');
+
+    // Choose a vocabulary type.
+    $radio = $this->getSession()->getPage()->find('named', ['radio', 'Tests the info alter hook']);
+    $radio->click();
+    $assert_session->assertWaitOnAjaxRequest();
+    $assert_session->elementExists('named', ['fieldset', 'Entity Test label bundle']);
+    $assert_session->checkboxNotChecked('Entity Test Bundle');
+
+    // Change the selected vocabulary type.
+    $radio = $this->getSession()->getPage()->find('named', ['radio', 'Entity test with bundle']);
+    $radio->click();
+    $assert_session->assertWaitOnAjaxRequest();
+    // Verify that the previous vocabulary reference handler form elements have
+    // been replaced with the ones of the newly selected handler.
+    $assert_session->elementNotExists('named', ['fieldset', 'Entity Test label bundle']);
+    $assert_session->elementNotExists('named', ['checkbox', 'Entity Test Bundle']);
+    $assert_session->elementExists('named', ['fieldset', 'Test entity bundle']);
+    $assert_session->checkboxNotChecked('Alpha');
+    $assert_session->checkboxNotChecked('Beta');
+    // Verify that at least a bundle must be specified.
+    $this->getSession()->getPage()->pressButton('Save');
+    $assert_session->pageTextContains('Error message Test entity bundle field is required.');
+
+    // Verify that the entity reference plugin ajax and states functionalities
+    // work correctly.
+    $sorty_by = $assert_session->fieldExists('Sort by');
+    // @todo Restore this line when Drupal 8.8 compatibility is dropped.
+    // $this->assertFalse($sorty_by->isVisible());
+    $this->getSession()->getPage()->findField('Alpha')->check();
+    $assert_session->assertWaitOnAjaxRequest();
+    $sort_by = $assert_session->fieldExists('Sort by');
+    $this->assertTrue($sort_by->isVisible());
+    $sort_by->selectOption('id');
+    $assert_session->assertWaitOnAjaxRequest();
+    $assert_session->fieldExists('Sort direction')->selectOption('DESC');
+
+    // Verify that the auto create field is not rendered.
+    $assert_session->fieldNotExists("Create referenced entities if they don't already exist");
+
+    // Submit the form.
+    $this->getSession()->getPage()->pressButton('Save');
+    $assert_session->pageTextContains('Status message Created new vocabulary Vocabulary 1.');
+
+    // Load the vocabulary config entity and verify that the auto create values
```

Comment:
`...and assert that all values have been saved correctly`

since you are checking for everything